frontend/src/pages/user-profile/user-profile.tsx

```diff
@@ -0,0 +1,11 @@
+import { UserProfileChatSidebar } from './components/components.js';
+
+const UserProfile: React.FC = () => {
+  return (
+    <>
+      <UserProfileChatSidebar />
+    </>
+  );
```

Comment:
Maybe let's do it this way?
```suggestion
  return <UserProfileChatSidebar />;
```

Reply:
it is one of the component on this page. so there will be in the future like div element istead of <></>

Reply:
If it will be changed anyway, I think it is better for now to keep it clear